apps/style/common.scss

```diff
@@ -1556,24 +1556,82 @@ $see-examples-thick-underline-color: $yellow;
   max-width: 100% !important;
 }
 
+a.send-to-phone-link {
+  text-decoration: none;
+  color: $light-gray;
+  &:hover {
+    color: $lightest-gray;
+    text-decoration: underline;
+  }
+}
```

Comment:
Would prefer if this was done as an inline style in the React component. Unfortunately, we don't have a way to handle things like hover that way right now (there are libraries that allow us to do that sort of thing, but we don't use them right now).

What do you think of naming this in a way that makes it super explicit that it's attached the the component, i.e. something like `WireframeSendToPhone_send-to-phone-link`?